.nspect-vuln-allowlist.toml

```diff
@@ -0,0 +1,39 @@
+version = "1.0.0"
```

Comment:
we need a license header here, can you add and check locally to pass the rat check?